C++/source/searching/Exponential_search.cpp

```diff
@@ -0,0 +1,66 @@
+// C++ program to find an element x in a
+// sorted array using Exponential search.
```

Comment:
Please read the [contribution guidelines](https://github.com/faheel/Algos/blob/master/CONTRIBUTING.md) to see how to format this comment.